packages/editor/src/components/url-popover/index.js

```diff
@@ -54,7 +54,7 @@ class URLPopover extends Component {
 					{ !! renderSettings && (
 						<IconButton
 							className="editor-url-popover__settings-toggle"
-							icon="ellipsis"
+							icon="arrow-down-alt2"
```

Comment:
Just noting, I think our panels actually use a material SVG here, but the IconButton doesn't support that yet, so I'm using the closest equivalent from Dashicons.